[docs][yba][PLAT-7657] IAM permissions for VPC (#18783)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md

- **VPC Setup**. Choose the VPC setup to use:
  - **Specify an existing VPC**. Select this option to use a VPC that you have created in AWS.

>   - **Create a new VPC**. Select this option to create a new VPC using YugabyteDB Anywhere. This option is considered beta and is not recommended for production use cases. If there are any classless inter-domain routing (CIDR) conflicts, using this option can result in a silent failure. For example, the following will result in a silent failure:

Reviewer: ```suggestion
  - **Create a new VPC**. Select this option to create a new VPC using YugabyteDB Anywhere. This option is considered Beta and is not recommended for production use cases. If there are any classless inter-domain routing (CIDR) conflicts, using this option can result in a silent failure such as the following:
```

Reviewer: make the change to stable as well.
